qt/place_page_dialog_user.cpp

```diff
@@ -22,6 +22,22 @@ namespace
 static int constexpr kMaxLengthOfPlacePageDescription = 500;
 static int constexpr kMinWidthOfShortDescription = 390;
 
+std::string getShortDescription(std::string description)
```

Comment:
If string_view is used, can it be passed as an argument, and returned from the function, to avoid two more allocations?

Reply:
Most likely can't return a view, since we need to add "..." at the end. But passing description as a const ref to this function would remove one more allocation.

Reply:
Modified `getShortDescription` to receive a const string ref.

(@biodranik feel free to resolve this conversation if you're fine with this)